README.md

```diff
@@ -18,9 +18,9 @@ Discuss on:
 * Next Scheduled Meeting: November 12 2019  16:00 UTC/9:00 am Pacfic/Noon Eastern [dial-in details](https://github.com/operator-framework/community/projects/2#card-27939134)
   * [Proposed Agenda](https://github.com/operator-framework/community/projects)
   * [Propose an agenda topic propose](https://github.com/operator-framework/community/projects/2#column-6841651)
-  
-### Operator SDK 
 
-* Wednesdays at 10:00 PT (Pacific Time) (monthly - fourth Wednesday every month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00&tz=PT%20%28Pacific%20Time%29).
+### Operator SDK
+
+* Wednesdays at 09:00 PT (Pacific Time) (monthly - first Wednesday of the month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00&tz=PT%20%28Pacific%20Time%29).
```

Comment:
For me shows: 

<img width="865" alt="Screenshot 2019-11-19 at 17 10 49" src="https://user-images.githubusercontent.com/7708031/69169128-bcd8ae80-0aef-11ea-95d1-24376bfd0859.png">

Note 09:00 PT (Pacific Time)

Shows that the `(http://www.thetimezoneconverter.com/?t=10:00&tz=PT%20%28Pacific%20Time%29).` needs to be replaced for `(http://www.thetimezoneconverter.com/?t=09:00&tz=PT%20%28Pacific%20Time%29).`

Reply:
To make this slightly easier, here's a GitHub suggestion:
```suggestion
* Wednesdays at 09:00 PT (Pacific Time) (monthly - first Wednesday of the month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=9:00&tz=PT%20%28Pacific%20Time%29).
```